client/src/components/CatsList/CatList.tsx

```diff
@@ -0,0 +1,85 @@
+import { Fragment, useEffect, useCallback, useState, useRef } from 'react';
+import { useCats } from '../../contexts/cats.context';
+import { useStyles } from './style';
+import { Cat } from './components/Cat';
+import { FilterInput } from '../common/components/FilterInput';
+import { NavLink } from 'react-router-dom';
+import { Button } from '../common/components/Button';
+import type { TCat } from '../../types/cat';
+import debounce from 'lodash.debounce';
+import axios from 'axios';
+
+const PLACE_HOLDER_TEXT: string = 'Search for a cat or mouse';
+const NO_CATS_FOUND: string = 'No Cats Found';
+const ADD_CAT_TEXT: string = 'Add Cat';
```

Comment:
That's nice you want to help me understand, but I can tell those are string
```suggestion
const PLACE_HOLDER_TEXT = 'Search for a cat or mouse';
const NO_CATS_FOUND = 'No Cats Found';
const ADD_CAT_TEXT = 'Add Cat';
```